actionview/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   `chomp!` both `url_string` and `request_uri` to fix issue with scopes and engines
```

Comment:
@rafaelfranca thank you! Fill fix in a second. 
First pull request, a little bit nerveus 😟